docs/source/examples/Notebook/Working With Markdown Cells.ipynb

```diff
@@ -161,6 +161,7 @@
     "Courtesy of MathJax, you can include mathematical expressions both inline: \n",
     "$e^{i\\pi} + 1 = 0$  and displayed:\n",
     "\n",
+    "Make sure you call %%latex before the statement \n",
```

Comment:
This whole notebook is about markdown cells. The `%%latex` cell magic is a shortcut to make latex output from *code* cells, which is a separate topic. In Markdown cells, you can write latex surrounded by `$` signs, as the examples already show.